iabtcf-extras-jackson/src/test/java/com/iabtcf/extras/jackson/gvl/VendorTest.java

```diff
@@ -112,6 +112,6 @@ public void testGetOverflow() {
 
     @Test
     public void testIsDeleted() {
-        Assert.assertFalse(vendorEight.isDeleted());
+        Assert.assertTrue(vendorEight.isDeleted());
```

Comment:
Is this change related to something else?

Reply:
a failing test

Reply:
Interesting, I wonder how I didn't notice this earlier.